### ISSUE TITLE
Searching by betterbibtexkey only looks in personal library, not group libraries.

### ISSUE THREAD

**Issue by cronco** [egh/zotxt#32]
This is visible in the [`findByBBTkey`](https://github.com/egh/zotxt/blob/984b073cb94f748eebec72f3af35475c90d70717/extension/content/modules/Core.jsm#L215) function.

**Comment by egh** [comment]
Yes, I'm not sure what the correct behavior would be here. I guess we need some sort of flag on the endpoint as to which libraries to search.

**Comment by cronco** [comment]
@egh Agreed, a flag on the endpoint would make sense. As a caller to the endpoint, I would expect default behaviour (i.e. with no flag passed) to search through all libraries.
